docs/topics/practices.rst

```diff
@@ -35,12 +35,17 @@ Here's an example showing how to run a single spider with it.
         ...
 
     process = CrawlerProcess({
-        'USER_AGENT': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)'
+        'USER_AGENT': 'Mozilla/4.0 (compatible; MSIE 7.0; Windows NT 5.1)',
```

Comment:
What about removing `USER_AGENT` as part of these changes? I believe the point of `USER_AGENT` was make it obvious that the first parameter of `CrawlerProcess` is a dictionary of settings, which the new settings already accomplish.

Reply:
maybe we can add CrawlerProcess(settings={}) so that it's totally clear.